### ISSUE TITLE
a lot of wasted space when on vertical monitor

### ISSUE THREAD

**Issue by alepocalypse** [continentaldivide/remix-checklist#25]
how it looks on my vertical monitor.  Might want to check aspect ratio before assigning the 1/3rd of screen columning

https://imgur.com/a/q6lGfGH

**Comment by continentaldivide** [comment]
Hey @alepocalypse!  Thanks for stopping by the repo and for the feedback.  I hadn't considered how the page might look on a vertical monitor, so this is helpful.

I've open-minded to tweaking the layout to use some of that space, perhaps by allowing wider viewports to have multiple columns side by side.  For now though it's a pretty low priority compared to adding actual features, plus it's partially contingent on whether/how I end up deciding to display some additional info per item.

This last month has been unexpectedly busy for me, which is disappointing since I anticipated being able to continue throwing a bunch of work at the site and instead have had an agonizingly small amount of time to spend on it.  Nevertheless I'll continue moving forward as much as I can, and I'll keep your feedback in mind as I dig in on some future changes.  Thanks again for sharing input.  🙂

Hope you're loving Remix so far!
